CMakeLists.txt

```diff
@@ -221,7 +229,7 @@ if (NMEA_UNIT_TESTS)
     # NMEA unit tests.
     #
     add_executable(utests-nmea src/nmea/parser.c tests/unit-tests/test_nmea_helpers.c)
-    target_link_libraries(utests-nmea dl)
+    target_link_libraries(utests-nmea ${CMAKE_DL_LIBS})
```

Comment:
Looks like line also needs the same `if(UNIX)` treatment as the ones above.

Reply:
Fixed in 4b61f67